CHANGELOG.md

```diff
@@ -12,10 +12,51 @@ Bug Fixes
   ChromaticObjects and image-setup keyword arguments (#683)
 - Added ability to manipulate the width of the moment-measuring weight function
   for the KSB shear estimation method of the galsim.hsm package. (#686)
+- Fixed bug in the (undocumented) function COSMOSCatalog._makeSingleGalaxy,
+  where the resulting object did not set the index attribute properly. (#694)
 
 
 New Features
-------------
-
+<<<<<<< HEAD
+- Added OutputCatalog class (#301, #691)
```

Comment:
say (briefly) what this is, for those who might find it useful but weren't necessarily looking at those issues?